AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Core/AWSAPICategoryPluginTests.swift

```diff
@@ -175,7 +175,7 @@ class AWSAPICategoryPluginTests: XCTestCase {
 
```

Comment:
please add the expectation that the publisher is finished for the test `testStorageEngineStartStopStart` (not the test at this comment, but above it)